media/css/newsletter/newsletter-developer.scss

```diff
@@ -5,26 +5,41 @@
 @import '../pebbles/includes/lib';
 
 .section-intro {
-    background: url('/media/img/newsletter/developer/heart.svg') top 30px center no-repeat,
-                url('/media/img/newsletter/developer/grid.svg') top left repeat,
-                #0c99d5;
-    @include background-size(200px 199px, 60px 60px);
-    height: 230px;
+    background: url('/media/img/newsletter/developer/8bit_hearts.svg') top 30px center no-repeat,
+                url('/media/img/newsletter/developer/bg_image.png') top left no-repeat,
+                #000;
+
+    @include background-size(66px 44px, auto auto);
+    background-position: top 75px center, top center;
```

Comment:
Sorry, should have noticed before... this rule is overriding the position included in the `background` rule above. We can either add these positioning values to the original `background` rule or remove the position there and only declare it here. Either way, no need to declare it twice.